tfjs-backend-webgpu/src/kernels/pad_webgpu.ts

```diff
@@ -42,19 +42,21 @@ export class PadProgram implements WebGPUProgram {
     this.dispatch = computeDispatch(
         this.dispatchLayout, this.outputShape, this.workGroupSize,
         [this.workPerThread, 1, 1]);
-
+    paddings.map((_, i) => this.uniforms += ` ivec2 pad${i};`);
     this.xShape = xShape;
     this.paddings = paddings;
```

Comment:
LGTM.
nit: this.paddings = paddings.map((_, i) => this.uniforms += ` ivec2 pad${i};`);?

Reply:
I don't need to change the value of `this.paddings`. I suppose above assignment will change the content of `this.paddings`.